css-layout-classes/styles.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-fourth {
+  width: calc(100% / 4);
+  display: flex;
```

Comment:
you can remove display flex here as you don't need to control the child of the column class in this instance